libr/util/buf.c

```diff
@@ -287,6 +287,22 @@ R_API int r_buf_set_bytes(RBuffer *b, const ut8 *buf, ut64 length) {
 	if (!(b->buf = malloc (length)))
 		return false;
 	memmove (b->buf, buf, length);
+
+	b->length = length;
+	b->empty = 0;
+	return true;
+}
+
+R_API int r_buf_set_bytes_steal(RBuffer *b, const ut8 *buf, ut64 length, bool steal_ptr) {
```

Comment:
I started by changing the sig for the original `r_buf_set_bytes`, and kept this when I decided it should be separate. wanted to know what you think. I'll remove it.